db.go

```diff
@@ -392,19 +393,13 @@ func (db *DB) Merge() error {
 				if err == ErrIndexOutOfBound {
 					break
 				}
-				_ = f.rwManager.Release()
```

Comment:
When the file is finished reading or an error occurs, you need to release the fd, right?